Distraction Shield/classes/BlockedSiteList.js

```diff
@@ -0,0 +1,88 @@
+
+function BlockedSiteList() {
+    this.list = [];
+
+    this.setList = function(blockedSiteArr) { this.list = blockedSiteArr; };
+    this.getList = function() { return this.list; };
```

Comment:
[https://github.com/RUGSoftEng/2017-The-Distraction-Shield/blob/master/Distraction%20Shield/modules/statistics/interception.js#L44](url)

here at least